Further improvements to README (#25)
Showmax/django-admin-page-lock: README.md

## Django Admin Page Lock

Page Lock for Django Admin allows developers to implement customizable locking of pages.

> With Admin Page Locking, only the designated (typically first) user receives his regular

Reviewer: ```suggestion
With Admin Page Locking, only the designated (typically first) user receives their regular
```

permissions assigned, including edit permissions (if available for the user).

> Subsequent users only get limited permissions from those assigned to him, making

Reviewer: ```suggestion
Subsequent users only get limited permissions compared to those assigned to the first user, making
```

```

#### Hiding specific html items
Add the class `page_lock_block` to any html tag you want to hide from users that are not

> currently the owner of the lock. One common usage for this feature is to hide submit

Reviewer: ```suggestion
currently holding the lock. One common usage for this feature is to hide submit
```

> buttons for users that are not the owners of the lock, for example:

Reviewer: ```suggestion
buttons for users that are not holding the lock, for example:
```
